.Dockerignore

```diff
@@ -0,0 +1,3 @@
+ui/node_modules
```

Comment:
I think the filename needs to be lowercase. It probably works on macOS because the FS is case insensitive...